data/avro-schema/src/main/resources/avro/net/corda/data/persistence/FIndWithNamedQuery.avsc

```diff
@@ -0,0 +1,21 @@
+{
+  "type": "record",
+  "name": "FindWithNamedQuery",
```

Comment:
I think it's worth adding the offset and limit parameters too. I know we have 2 JIRAs splitting it up, but adding them now will avoid another API change that then needs a version bump.